apps/Client/src/pages/Login/LoginPage.tsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 import { useForm } from 'react-hook-form';
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
-// Import your logo and button here
 import riderPhoto from "../../pages/Auth/Images/image 4.png";
 import Button from '../../components/Button';
 import riderLogo from "../../pages/Auth/Images/Logo.png";
 
 const LoginPage: React.FC = () => {
   const { register, handleSubmit } = useForm();
 
-  const onSubmit = (data: unknown) => {
```

Comment:
Using unknown type is not advisable when using typescript. Try finding a way to fix the code without using unknown type. I think Chat can help out with it.